jstreegrid.js

```diff
@@ -213,6 +213,10 @@
 				for (i=0;i<cols.length;i++) {
 					// create the column
 					$("<div></div>").addClass("jstree-default jstree-grid-column jstree-grid-column-"+i+" jstree-grid-column-root-"+this.rootid).appendTo(this.midWrapper);
+					
+					if (typeof(cols[i].value) === "function") {
+						console.warn("[jstree-grid] using value as a function is deprecated, use 'format' option instead.");
+					}
 				}
```

Comment:
Let's remove the deprecation warning. We are OK with no longer having it documented - it is a poor man's way of seeing who complains without breaking anything, see who really needs it, quite smart on your part - but the deprecation warning is a bit much.